templates/student/student.html

```diff
@@ -14,13 +14,16 @@
 <hr />
 
 <table class="table table-striped table-bordered">
-    {% for sol in solutions %}
+    {% for sol in solutions|dictsort:"programming_language" %}
```

Comment:
I think we shouldn't dict sort the solutions on the student page.  Just show them in order of the week.